catalog/spatial/geocoding/spatial-geocoding-lucene/pom.xml

```diff
@@ -78,10 +92,17 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Export-Package>org.codice.ddf.spatial.geocoding.index</Export-Package>
+                        <Private-Package>
+                            org.apache.lucene.sandbox.queries,
```

Comment:
Might want to have a comment explaining why this is being private-packaged, based on the earlier discussion.